### PR TITLE
Fixes gamebreaking syndicakes bug.

### LINKED ISSUE #41591: Hunger slowdown as ops
## Round ID: 97310

This round, I got hunger slowdown just 15 minutes into the round as nuke op. There is a single slice of pizza at the op base, as well as some donuts on the ship. Regardless, I think ops should start with a higher nutrition so you don't get this dumb shit because you forgot to eat or there was not enough food for everyone. The 20 minute war timer is important to keep in mind here as well.

Also, I ded.

### PR DESCRIPTION
**Pull request by nemvar**
## About The Pull Request

How is the syndicake ever supposed to be successful if there aren't enough syndicakes in the syndicake base?

## Why It's Good For The Game

Fixes #41591

## Changelog
:cl:
balance: Syndicake buff
/:cl:

### CODE REVIEW

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -8680,11 +8680,13 @@
 /area/centcom/ferry)
 "te" = (
 /obj/structure/table/wood,
-/obj/item/reagent_containers/food/snacks/pizzaslice/mushroom,
```
Comment (optimumtact) [inline _maps/map_files/generic/CentCom.dmm]: my mushroom pizza

Comment (nemvar) [reply _maps/map_files/generic/CentCom.dmm]: the future is now.

**Comment by Shadowflame909** [comment]
Pepperoni and Canadian Bacon is the better combo

File: _maps/map_files/generic/CentCom.dmm (hunk 1 above)
Comment (PKPenguin321) [reply _maps/map_files/generic/CentCom.dmm]: why not both

Comment (nemvar) [reply _maps/map_files/generic/CentCom.dmm]: wtf powercreep wtf

**Pull request merged**